docs/windows.md

```diff
@@ -0,0 +1,74 @@
+- Make the deployment also Windows native. A Windows sysadmin should be able to apply their knowlege hand habits as much as possible.
+- Make the deployment functionally equivalent (so far as possible) to the linux deployment.  A DC/OS UI user should not be able to obviously tell the difference between a linux and windows host without looking at the host attributes.
+
+The installation adds to those goals:
+- The deployment should be feasible on a Windows host without additional preparation. As a result, the deployment must use only tools available on a freshly installed windows machine.
+
+## changes to the dcos/dcos repository
+
+The package tree, top level scripts, and configuration files are, as much as possible, organised in a parallel manner to the
+linux build process, but they are not the same, so shell scripts are shadowed by powershell scripts of the same name, ie build_local becomes build_local.ps1, prep_local becomes prep_local.ps1 and so on.  
+
+- The package tree is paralleled by packages.windows. 
+  - In each directory we have a buildinfo.json parallel to the linux buildinfo.json, but usually different as windows requires different dependencies, packages, git repositories or branches, and commit shas from the linux equivelant. 
+  - The build script located in each of the pacakge directories is paralleled by a build.ps1 script which is functionally similar.
+  - In the packages.windows tree there are setupinfo.json and setup.ps1 scripts wwhich provide setup information at package 
+    install time. Those files get included in the tar.xz build artifact.
+    
+- Changes to the python code in pkgpanda, release, gen, dcos_installer, ssh and related directories are
+  - is_windows variable so that all os-specific branches don't have to go through a string compare
+  - replacement of shell check_calls to if is_windows: check_call <windows> else: check_call <linux>
+  - added constants for paths so that the windows and linux paths need not require inline "if is_windows" sections.
+
+- A parallel set of yaml tmeplates.  These files are specialised by gen in the same manner as the windows version. The yaml
+  templates are used to create processed configuration files for a particular build and deployment. The windows versions will i
+  be pretty much line by line different owing to the difference in the two OS environments. For example, all paths will be
+  different, the syntax of environment setting is different, and windows has its own service manager making systemd units not
+  applicable to the windows build.  In place of systemd unit files, we will have a number of powershell scripts to implement
+  the same functionality so far as practical.
```

Comment:
system configuration on windows is quite different form configuration on Linux. A lot of these yaml files are either configuration or bash commands. again, these do not work on windows.
because these are so different we cannot really reuse them. They are there for now as a place holder for the final work that is still in place.